cmd/agentctl2/cmd/show_version_cmd.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// RootCmd represents the base command when called without any subcommands.
+var showVersion = &cobra.Command{
+	Use:     "version",
+	Aliases: []string{"V"},
+	Short:   "Show agent version",
+	Long: `
+	Show agent version
+`,
+	Run: versionFunc,
+}
+
+func init() {
+	RootCmd.AddCommand(showVersion)
+}
+
+func versionFunc(cmd *cobra.Command, args []string) {
+	fmt.Printf("agentctl version 0.1\n")
```

Comment:
The version of agent can be retrieved from `ligato/cn-infra/agent` package. 

See how the vpp-agent prints it at boot: https://github.com/ligato/vpp-agent/blob/731e9b4b9b2e37407e805221dcab9ccbd0482353/cmd/vpp-agent/main.go#L41